src/main/kotlin/no/nav/tilleggsstonader/sak/opplysninger/oppgave/OppgaveService.kt

```diff
@@ -136,6 +136,11 @@ class OppgaveService(
                 saksbehandler = saksbehandler,
                 versjon = versjon,
             )
+        val finnOppgave = oppgaveRepository.findByGsakOppgaveId(gsakOppgaveId)
+
+        finnOppgave?.let { oppgave ->
```

Comment:
Burde vi kaste en feil hvis oppgaven ikke finnes når man skal fordele oppgave? Ser for meg at det er en state man ikke burde komme i?